test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ValidateTests.cs

```diff
@@ -134,9 +137,11 @@ public async void GivenAValidateRequest_WhenTheResourceIsInvalid_ThenADetailedEr
                     expression);
         }
 
-        [Fact]
+        [SkippableFact]
         public async void GivenAValidateByIdRequest_WhenTheResourceIsValid_ThenAnOkMessageIsReturned()
         {
+            Skip.If(ModelInfoProvider.Instance.Version == FhirSpecification.R5, "Validation not currently working correctly for R5");
```

Comment:
A little scared of doing this. It isn't an R5 thing, its a Firely package thing. Can we skip if Hl7.FHIR packages <= version x.x.x?